src/main/java/com/sleekbyte/tailor/listeners/KPrefixListener.java

```diff
@@ -28,8 +28,12 @@ public void enterTopLevel(TopLevelContext topLevelCtx) {
         List<IdentifierContext> names = DeclarationListener.getConstantNames(topLevelCtx);
         names.forEach(
             ctx -> {
-                String constantName = ctx.getText();
+                String constantName = CharFormatUtil.unescapeIdentifier(ctx.getText());
                 Location location = ListenerUtil.getContextStartLocation(ctx);
+                // Ensure that the violation column number reports the character after the opening backtick.
+                if (CharFormatUtil.isEnclosedInBackticks(ctx.getText())) {
```

Comment:
We could create a function that returns the first character of an identifier and accounts for the escaped case (since we're doing this in a couple of places).